integration-test/test.bats

```diff
@@ -101,22 +130,35 @@
 
 @test "govvv build - reads Version from ./VERSION file" {
     tmp="${BATS_TMPDIR}/a.out"
-    run bash -c "cd ${BATS_TEST_DIRNAME}/app-versioned && govvv build -o ${tmp} ."
+    run bash -c "cd ${BATS_TEST_DIRNAME}/app-versioned && govvv build -o ${tmp}"
     echo "$output"
     [ "$status" -eq 0 ]
 
     run "$tmp"
     echo "$output"
     [ "$status" -eq 0 ]
-    [[ "$output" == "Version=2.0.1" ]]
+    [[ "$output" == "Version=2.0.1-app-versioned" ]]
+}
+
+@test "govvv build - ./VERSION file overridden by -version option" {
+    tmp="${BATS_TMPDIR}/a.out"
+    run bash -c "cd ${BATS_TEST_DIRNAME}/app-versioned && govvv build -o ${tmp} -version 1.2.3-command-line"
+    echo "$output"
+    [ "$status" -eq 0 ]
+
+    run "$tmp"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    [[ "$output" == "Version=1.2.3-command-line" ]]
 }
 
 @test "govvv compiled with govvv" {
-    run govvv install -a
+    run govvv build
     echo "$output"
     [ "$status" -eq 0 ]
 
-    run govvv
+    run ./govvv
```

Comment:
do not change this. $GOPATH/bin should be in your path.

Reply:
I changed it to pick up the artifact built by `govvv build`.